Content.Server/Salvage/SpawnSalvageMissionJob.cs

```diff
@@ -279,7 +285,9 @@ protected override async Task<bool> Process()
         return true;
     }
 
-    private async Task SpawnRandomEntry(MapGridComponent grid, IBudgetEntry entry, Dungeon dungeon, Random random)
+    private async Task SpawnRandomEntry(EntityUid gridUid, MapGridComponent grid, IBudgetEntry entry, Dungeon dungeon,
```

Comment:
per conventions this should be made into an `Entity<MapGridComponent`, not two separate params